tests/standalone_activity_test.go

```diff
@@ -164,10 +165,48 @@ func (s *standaloneActivityTestSuite) TestIDConflictPolicy_FailsIfExists() {
 		},
 		StartToCloseTimeout: durationpb.New(1 * time.Minute),
 	})
+
 	require.Error(t, err)
+	statusErr := serviceerror.ToStatus(err)
+	require.Equal(t, codes.AlreadyExists, statusErr.Code())
+
+	var details *errordetailspb.ActivityExecutionAlreadyStartedFailure
+	for _, detail := range statusErr.Details() {
+		if d, ok := detail.(*errordetailspb.ActivityExecutionAlreadyStartedFailure); ok {
+			details = d
+			break
+		}
+	}
+	require.NotNil(t, details, "expected ActivityExecutionAlreadyStartedFailure in error details")
+	require.Equal(t, s.tv.RequestID(), details.StartRequestId)
+	require.Equal(t, startResponse.GetRunId(), details.RunId)
 }
 
-// TODO(fred): add test for BusinessIDConflictPolicyUseExisting after rebasing on main
+func (s *standaloneActivityTestSuite) TestIDConflictPolicy_UseExistingNoError() {
```

Comment:
nit: How about we group together all the conflict policy tests as subtests of `TestIDConflictPolicy`? I've been doing that for e.g. heartbeat tests.